mappings/net/minecraft/stat/RegisteringStat.mapping

```diff
@@ -0,0 +1 @@
+CLASS yd net/minecraft/stat/RegisteringStat
```

Comment:
This sucks!!! Idk what to name it

Name this `Stat` and rename `Stat` to `StatBase`?